lib/net.c

```diff
@@ -1517,18 +1517,16 @@ start_port(uint8_t port_id, uint8_t *pnum_succ_ports,
 	 * are activated on a timer, so this can take some time.
 	 */
 	do {
-		rte_eth_link_get(port_id, &link);
-
+		ret = rte_eth_link_get(port_id, &link);
 		/* Link is up. */
-		if (link.link_status)
```

Comment:
Style: preserve the blank line after line `ret = rte_eth_link_get(port_id, &link);`, so it's clear that the comment `/* Link is up. */` refers to the 3 lines below it.